de.fu_berlin.inf.dpp/saros.properties

```diff
@@ -41,7 +41,7 @@ de.fu_berlin.inf.dpp.server.SUPPORTED = false
 saros.swtbrowser=false
 
 ################### Feedback ############################
-
+de.fu_berlin.inf.dpp.feedback.ENABLED = false
```

Comment:
No comment about modifying the plugin.xml to fully activate that feature is bad.

Reply:
Thanks, I will add the comment